config/crd/bases/mellanox.com_nicclusterpolicies.yaml

```diff
@@ -138,15 +150,19 @@ spec:
                 - version
                 type: object
               secondaryNetwork:
-                description: SecondaryNetwork describes configuration options for
-                  secondary network
+                description: SecondaryNetworkSpec describes configuration options
```

Comment:
it called secondaryNetwork so why you change it to SecondaryNetworkSpec?  maybe we can just remove it and write "describes configuration options .."

Reply:
Need to find a proper fix to make both kubebuilder and godoc happy